one-time.js

```diff
@@ -21,17 +21,20 @@ function updatePermissionsApiStatus(permissionName, permissionStatus) {
 }
 
 // Display the feature access status (whether the feature can actually be accessed successfully in the browser)
-function updateAccessStatus(permissionName, accessStatus) {
+function updateAccessStatus(permissionName, accessStatus, message) {
   const textToDisplay = API_ACCESS_STATUSES[accessStatus];
   document.querySelector(`#${permissionName}-access-status`).innerText =
     textToDisplay;
+  if (!!message) {
```

Comment:
I think a direct check would be more idiomatic.
```suggestion
  if (message) {
```